hardware/config/default.xml

```diff
@@ -14,7 +14,7 @@
       </params> -->
     </CmpDev>
   </CmpDevs>
-  <bus burstLength="4" writeCombine="false" mmu="false" />
+  <bus burstLength="4" writeCombine="false" mmu="false" roundRobinArbiter="false" />
```

Comment:
Please do not change the default.xml. This would break all design. Change it in the project XML file.